### PR TITLE
Implement simple negate operation

### PR DESCRIPTION
**Pull request by maurobringolf**
#1

### CODE REVIEW

File: src/interpreter/kernel/exec.js [hunk 1]
```diff
@@ -991,6 +991,37 @@ export function executeStackFrame(frame: StackFrame, depth: number = 0): any {
         throw new RuntimeError('Unsupported operation ' + instruction.id + ' on ' + instruction.object);
       }
 
+      break;
```
Comment (xtuc) [inline src/interpreter/kernel/exec.js]: Btw do you know if there a rule to ensure the break in ESLint? Sometimes I forget it and loose a lot of time trying to debug it.

File: test/interpreter/fixtures/fneg/exec.tjs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  assert.typeOf(res, 'number');
+  assert.equal(res, -0.5);
+});
+
+it('should negate negative input', () => {
+  const res = m.exports.neg(-12341234);
+
+  assert.typeOf(res, 'number');
+  assert.equal(res, 12341234);
```
Comment (xtuc) [inline test/interpreter/fixtures/fneg/exec.tjs]: The result type should be a float?

Comment (maurobringolf) [reply test/interpreter/fixtures/fneg/exec.tjs]: Not sure what you mean, integers are floats?

File: src/interpreter/kernel/exec.js (hunk 1 above)
Comment (maurobringolf) [reply src/interpreter/kernel/exec.js]: I don't think so https://github.com/eslint/eslint/issues/7697

File: test/interpreter/fixtures/fneg/exec.tjs (hunk 2 above)
Comment (xtuc) [reply test/interpreter/fixtures/fneg/exec.tjs]: I expected 12341234.0 but you are right.

Comment (maurobringolf) [reply test/interpreter/fixtures/fneg/exec.tjs]: Ah yeah these are the same thing in JS

**Pull request merged**